ui/src/app/cohort-search/attributes-page/attributes-page.component.tsx

```diff
@@ -315,8 +315,7 @@ export const AttributesPage = withCurrentWorkspace() (
         }
         return acc;
       }, formErrors);
-      formValid = formValid ||
-        (this.isMeasurement && !operatorSelected && form.cat.some(attr => attr.checked));
+      formValid = formValid && !(this.isMeasurement && !operatorSelected && !form.cat.some(attr => attr.checked));
```

Comment:
can we add a comment explaining what case this code is validating? `!(this.isMeasurement && !operatorSelected && !form.cat.some(attr => attr.checked))`

Reply:
Good idea, done. Having these domain-specific conditions can get confusing.